test/libs/metrics/top_level_metrics_test.py

```diff
@@ -1,40 +1,90 @@
 import dataclasses
-import io
 
 import numpy as np
 import pandas as pd
 import pytest
 import structlog
+from freezegun import freeze_time
 from covidactnow.datapublic.common_fields import CommonFields
-from covidactnow.datapublic import common_df
+
 from api import can_api_v2_definition
 from libs.metrics import top_level_metrics
 from libs.datasets.timeseries import MultiRegionDataset
 from libs.datasets.timeseries import OneRegionTimeseriesDataset
 from libs.pipeline import Region
-from freezegun import freeze_time
 
+from test.libs.datasets import timeseries_test
 from test.dataset_utils_test import read_csv_and_index_fips_date
 
+INPUT_COLUMNS = [
```

Comment:
This is used by one test and passed directly into _build_timeseries_dataframe().  Would it make sense to just move this inside of _build_timeseries_dataframe()?  Else, can you add some comments / TODOs to make it clearer how you expect this to be evolved in the future?